apps/laboratory/tests/shared/pages/WalletPage.ts

```diff
@@ -7,18 +7,24 @@ export class WalletPage {
   private readonly baseURL = WALLET_URL
 
   private readonly gotoHome: Locator
+  private readonly vercelPreview: Locator
 
   constructor(public readonly page: Page) {
     this.gotoHome = this.page.getByTestId('wc-connect')
+    this.vercelPreview = this.page.locator('css=vercel-live-feedback')
   }
 
   async load() {
     await this.page.goto(this.baseURL)
   }
 
   async connect() {
+    const isVercelPreview = (await this.vercelPreview.count()) > 0
+    if (isVercelPreview) {
+      // eslint-disable-next-line  @typescript-eslint/no-explicit-any
+      await this.vercelPreview.evaluate((iframe: any) => iframe.remove())
+    }
```

Comment:
seems like repeated code, could this be abstracted to a util/helper ?
also, doesn't `HTMLIFrameElement` work?
something like 

```ts
export const removeVercelPreview = async (page: Page) => {
    const vercelPreviewCount = await this.vercelPreview.count()
    if (vercelPreviewCount > 0) {
      await this.vercelPreview.evaluate((iframe: HTMLIFrameElement) => iframe.remove())
    }
}```

Reply:
I'm copying it twice in test code. So my rule is you can copy twice then you need to refactor. Hope this is cool since it's tests and they will need some love anyways when web3wallet fixes are live.

Thanks on `HTMLIFrameElement`. Added!